Client/Extensions/UIImageViewExtensions.swift

```diff
@@ -41,15 +41,12 @@ extension UIImageView {
     // This is a helper function for custom async loaders. It starts an operation that will check for the image in
     // a cache (either one passed in or the default if none is specified). If its found in the cache its returned,
     // otherwise, block is run and should return an image to show.
-    private func runBlockIfNotInCache(key: String, var cache: SDImageCache? = nil, completed: CompletionBlock, block: () -> UIImage?) {
+    private func runBlockIfNotInCache(key: String, cache: SDImageCache, completed: CompletionBlock, block: () -> UIImage?) {
```

Comment:
To retain the same behaviour shouldn't we set the default param for SDImageCache to be the shared instance?